distribution/lib/Standard/AWS/0.0.0-dev/src/Database/Redshift/Internal/Redshift_Dialect.enso

```diff
@@ -139,6 +152,16 @@ type Redshift_Dialect
     supports_float_round_decimal_places : Boolean
     supports_float_round_decimal_places self = False
 
+    ## PRIVATE
+       Specifies whether the Database supports CREATE TEMPORARY TABLE syntax.
+    suppports_temporary_table_syntax : Boolean
+    suppports_temporary_table_syntax self = True
+
+    ## PRIVATE
+       Specifies whether the Database supports CREATE TABLE #temptablename syntax.
+    suppports_temporary_with_hash_syntax : Boolean
+    suppports_temporary_with_hash_syntax self = False
```

Comment:
Again, it feels to me like this is a bit of duplication. Is there a database that actually supports both syntaxes?

I'd prefer to have `temporary_table_syntax -> Temporary_Table_Syntax` where that type could have e.g. ~3 variants: `Temporary_Keyword`, `Hash_Syntax`, `Not_Supported`.

Reply:
Also, I don't see `suppports_temporary_with_hash_syntax` used anywhere? I assume this is deliberate and will come in some future PR?

Reply:
Removed suppports_temporary_with_hash_syntax for now. Will refactor as part of https://github.com/enso-org/enso/issues/10711